src/globals.js

```diff
@@ -7,4 +7,19 @@ export const clearTimeout = global.clearTimeout;
 
 export const document = window && window.document;
 export const navigator = window && window.navigator;
-export const sessionStorage = window && window.sessionStorage;
+
+let defined = {
+	sessionStorage: ( function() {
```

Comment:
Let's just export the result directly:
```js
export const sessionStorage = ( function() { ... }() );
```
No need to have an intermediate value.